### PR TITLE
First pass at travis-ci build, fixed RTD build problem

### PR DESCRIPTION
**Pull request by jxstanford**
The commit establishes a travis-ci build.  Currently, the build set up
the build environment with postgres and elasticsearch, installs requirements,
runs pep8, and does a no-op command.  The command to execute python tests
is there, but currently we are waiting for a stable home for the ES test
data.  Pivotal chore created for finalization:
    https://www.pivotaltracker.com/story/show/97131160

The RTD build failures were also addressed here.  The issue was a missing
dependency on mkdocs, and organization of the markdown files.  They
are now placed under the 'docs' folder of the source tree.  Doc builds
appear to be working correctly.

Refs:
    - http://goldstone-server.readthedocs.org/en/travis
    - https://travis-ci.org/Solinea/goldstone-server

### CODE REVIEW

File: goldstone/tenants/templates/new_tenant_body_for_tenant_admin.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% load i18n %}{% autoescape off %}
+{% load i18n }{% autoescape off %}
```
Comment (stugots) [inline goldstone/tenants/templates/new_tenant_body_for_tenant_admin.txt]: Looks like the % was accidentally deleted.

Comment (jxstanford) [inline goldstone/tenants/templates/new_tenant_body_for_tenant_admin.txt]: fixed

**Comment by stugots** [comment]
:+1:

**Comment by lexjacobs** [comment]
:+1:

**Pull request merged**